daphne/src/roles_test.rs

```diff
@@ -25,6 +25,16 @@ use prio::codec::{Decode, Encode};
 use rand::{thread_rng, Rng};
 use std::{ops::DerefMut, vec};
 
+// MockAggregator's implementation of DapLeader::get_report() always returns reports for a single
+// task. This macro is used to conveniently unwrap the task ID and reports for testing purposes.
+macro_rules! get_reports {
+    ($leader:expr, $selector:expr) => {{
+        let reports_per_task = $leader.get_reports($selector).await.unwrap();
+        assert_eq!(reports_per_task.len(), 1);
```

Comment:
I wonder whether we really need this assertion here.

Reply:
As mentioned, this is here to validate an assumption that users of `get_reports!` make, which is that there is one and only one task.